test/experiments/test_unique_worker.py

```diff
@@ -0,0 +1,236 @@
+from unittest.mock import mock_open, patch
+
+import pytest
+
+from operationsgateway_api.src.experiments.unique_worker import (
+    assign_event_to_single_worker,
+    UniqueWorker,
+)
+
+
+class TestUniqueWorker:
+    # TODO - path doesn't seem to be mocking correctly
```

Comment:
Did you mean to leave this TODO in?

Reply:
I didn't, but I'm glad I did (and you spotted it)! The mocking was being ignored, but I've fixed this by mocking directly onto the class variable. Fixed in c861b49bbf35e931dd8b849155181d38ca7a320a